client/src/api/concerts.js

```diff
@@ -1,10 +1,11 @@
 export async function postConcert(concerts) {
-  const response = await fetch("http://localhost:4000/concerts", {
+  const response = await fetch("/api/concerts", {
     method: "POST",
     headers: {
       "Content-Type": "application/json",
     },
     body: JSON.stringify(concerts),
+    Encrypt: "true",
   });
   const createdConcert = await response.json();
   return createdConcert;
```

Comment:
Just a minor thing. I assume this POST method will let you post one concert, right? Then it might make sense to also change the parameter from `concerts` to `concert`.

Reply:
That makes sense! :) 
I'll keep that in mind for refactoring, because I have to change it in other places as well